metadata-models/src/main/pegasus/com/linkedin/metadata/snapshot/ModelSnapshot.pdl

```diff
@@ -0,0 +1,20 @@
+namespace com.linkedin.metadata.snapshot
+
+import com.linkedin.common.MLModelUrn
+import com.linkedin.metadata.aspect.ModelAspect
+
+/**
+ * Model Snapshot entity details.
+ */
+record ModelSnapshot {
```

Comment:
not sure if this comment has been raised before, but we could name it `MLModelSnapshot` and the aspect as `MLModelAspect`, similar to `MLFeatures`?